bin/generate

```diff
@@ -9,3 +9,12 @@ generators = Generator::CommandLine.new(paths).parse(ARGV)
 exit 1 unless generators
 
 generators.map(&:call)
+
+configletPath = "#{EXERCISM_RUBY_ROOT}/bin/configlet"
+unless File.file?(configletPath)
+  raise Exception.new("Unable to find configlet, fetch using bin/fetch_configlet")
+end
+
+generators.each do |generator|
+  system("#{configletPath} generate #{EXERCISM_RUBY_ROOT} --only=#{generator.exercise.slug}")
+end
```

Comment:
I'm not entirely sure if this is the right way to be approaching this problem, I'll have to think about it for a while.

Is README generation related to test suite generation?
Will we always want to do both at the same time?
Should the code for doing this live inside the `Generator` with the other 3 things that it does already?

Reply:
Do we want to be able to ONLY regenerate the readme for a problem?
Does this need to be controlled by command line arguments to bin/generate?